src/clj_sqs_extended/serdes.clj

```diff
@@ -0,0 +1,51 @@
+(ns clj-sqs-extended.serdes
+  (:require [cognitect.transit :as transit]
+            [tick.alpha.api :as t]
+            [cheshire.core :as json])
+  (:import [java.io ByteArrayOutputStream ByteArrayInputStream]))
+
+
+(def ^:private tick-time-writer
+  (transit/write-handler
+    "tick-time"
+    (fn [t] (.toString t))))
```

Comment:
looks like there's an outstanding PR upstream to enable serde for tick <> transit.  https://github.com/juxt/tick/issues/34#issuecomment-503641158, which points to https://github.com/henryw374/time-literals/issues/2

perhaps make use of their suggestion https://gist.github.com/jdf-id-au/2e91fb63ce396b722c1d6770154f1815#file-transit-connection-cljc-L54 as it handles all the tick classes